dev/service-registry/service-locator/src/main/scala/com/lightbend/lagom/gateway/AkkaHttpServiceGateway.scala

```diff
@@ -63,7 +64,7 @@ class AkkaHttpServiceGateway(coordinatedShutdown: CoordinatedShutdown, config: S
         }
       case NotFound(registryMap) =>
         log.debug("Sending not found response")
-        Future.successful(renderNotFound(request, path, registryMap))
+        Future.successful(renderNotFound(request, path, registryMap.map { case (k, v) => (k, v._2) }))
```

Comment:
Another change I introduced affects the API on the service registry. This had an impact on the service gateways. This is just refactoring from `InetAddress` to `URI`.

A side effect of this change is that we can now send back the `scheme` in the result while `InetAddress` only has room for the `host` and the `port`.

Reply:
I don't see the scheme is used anywhere.

Reply:
My comment above is missleading. This code is the client side of the service registry (which is the one sending back the `scheme`). I was assuming the `scheme` would be necessary when we use these improvements in #1443 but I'm not so sure anymore and it's possible we'll be able to go back to `InetAddress`.

When/if this goes back to consume an `InetAddress` then we'll have to assume the scheme (if needed) from the `portName` or the incoming `request`.